config/src/test/resources/org/springframework/security/config/http/InterceptUrlConfigTests-FilterAllDispatcherTypesFalse.xml

```diff
@@ -24,9 +24,9 @@
 			http://www.springframework.org/schema/beans
 			https://www.springframework.org/schema/beans/spring-beans.xsd">
 
-	<http auto-config="true" filter-all-dispatcher-types="false">
+	<http auto-config="true">
```

Comment:
Is this test file still meaningful? I ask because it is called "FilterAllDispatcherTypesFalse". Perhaps this file and its test need to be renamed or removed.